Reworded warning for clarity (#6620)
umbraco/UmbracoDocs: 14/umbraco-cms/tutorials/add-microsoft-entra-id-authentication.md

This tutorial takes you through configuring Microsoft Entra ID (Azure Active Directory/Azure AD) for the member login on your Umbraco CMS website.

{% hint style="warning" %}

> Entra ID (formerly Azure AD) conflicts with Umbraco ID when used for backoffice logins on Umbraco Cloud projects, though this does not affect member logins.

Author: ```suggestion
Entra ID (formerly Azure AD) conflicts with Umbraco ID when used for backoffice user logins on Umbraco Cloud projects, though this does not affect member logins.
```

> Entra ID (formerly Azure AD) conflicts with Umbraco ID when used for backoffice user logins on Umbraco Cloud projects, though this does not affect member logins.

Author: ```suggestion
Entra ID (formerly Azure Active Directory) conflicts with Umbraco ID when used for backoffice user logins on Umbraco Cloud projects, though this does not affect member logins.
```

> Microsoft Entra ID (formerly Azure Active Directory) conflicts with Umbraco ID when used for backoffice user logins on Umbraco Cloud projects. This does not affect member logins.

Author: ```suggestion
Microsoft Entra ID conflicts with Umbraco ID when used for backoffice user logins on Umbraco Cloud projects. This does not affect member logins.
```

> Microsoft Entra ID (formerly Azure Active Directory) conflicts with Umbraco ID when used for backoffice user logins on Umbraco Cloud projects, though this does not affect member logins.

Author: ```suggestion
Microsoft Entra ID (formerly Azure Active Directory) conflicts with Umbraco ID when used for backoffice user logins on Umbraco Cloud projects. This does not affect member logins.
```

> **We highly recommend not using Azure AD for backoffice authentication on your Umbraco Cloud projects**.

Author: ```suggestion
**We strongly recommend against using Entra ID for backoffice authentication on Umbraco Cloud projects**.
```

For member authentication, other [External Login Providers](../reference/security/external-login-providers.md), such as Google Auth and OpenIdConnect, are still supported and compatible with Umbraco Cloud.

> Additionally, the **Custom Identity Provider for Umbraco Cloud** feature is currently in development. Once released, it will enable seamless integration of your own Entra ID with the Umbraco Cloud Portal, providing enhanced user authentication. For more information, see the [Umbraco Roadmap](https://umbraco.com/products/knowledge-center/roadmap/).

Author: ```suggestion
```
